MonoGame.Framework/Platform/Native/Platform.Interop.cs

```diff
@@ -322,6 +334,25 @@ public static partial int Window_ShowMessageBox(
          string[] buttons,
          int count);
 
+    [LibraryImport(MonoGameNativeDLL, EntryPoint = "MGP_Window_GetTextInputState", StringMarshalling = StringMarshalling.Utf8)]
+    [return: MarshalAs(UnmanagedType.U1)]
+    public static partial bool Window_GetTextInputState(MGP_Window* window);
```

Comment:
What does this do?  It seems like it needs a better function name to answer this question.